.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    continue-on-error: false
+    strategy:
+      matrix:
+        php: ['7.2', '7.3', '7.4', '8.0', '8.1', '8.2']
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt-get install libcurl4-openssl-dev libxml2 libxml2-dev
```

Comment:
According to the pipeline only `libcurl4-openssl-dev` is needed
Also it should be `sudo apt-get install -y libcurl4-openssl-dev`. When I ran it locally it hanged since it didn't have `-y`